### ISSUE TITLE
REPL does not work when taiko is invoked as a runner

### ISSUE THREAD

**Issue by Debashis9012** [getgauge/taiko#822]
### Expected behavior
await repl();
repl should started

### Actual behavior
repl did not get start.
Getting below error message:
```
{ Error: Cannot find module 'taiko/recorder'
    at Function.Module._resolveFilename (module.js:548:15)
    at Function.Module._load (module.js:475:25)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/debas/dadebash/work/taikoProject/issue272/test.js:2:18)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3) code: 'MODULE_NOT_FOUND' }
```

### Steps to reproduce
run the below script with the below option `taiko --load sampleREPLtest.js` 

script:
```
const { openBrowser, goto, closeBrowser } = require('taiko');
const { repl } = require('taiko/recorder');
(async () => {
    try {
        await openBrowser();
        await goto("google.com");
        await repl();
        await write("taiko");
        await click("Google Search");
        await goBack();
    } catch (error) {
        console.error(error);
    } finally {
        await closeBrowser();
    }
})();

```

### Versions
```
326874da672b782e171f5d7c070e68137c6201c6
```

**Comment by BugDiver** [comment]
Fix should be available in 24fe11d94335790709d13b3f64ec046ac153a4a2

**Comment by Debashis9012** [comment]
Tested this above issue.
Observations:
1. once the control comes to repl if we do `.exit` then its showing the below user log
```
> .exit
 ✔ Browser closed
 ✔ Browser closed
```
2.  After doing `.exit` user should come out of repl. If we hit `.exit` once again then repl hangs up

<img width="482" alt="Screen Shot 2019-09-19 at 3 14 21 PM" src="https://user-images.githubusercontent.com/46309201/65233293-32c09980-daf0-11e9-87e8-d925695373d6.png">

Tested version: `24fe11d94335790709d13b3f64ec046ac153a4a2`

**Comment by BugDiver** [comment]
Fix should be available in c9fdf29

**Comment by Debashis9012** [comment]
This issue has been verified and found fixed.
Tested version: `c9fdf2959cce1b94e6ed6964aecaca505c5cf43d`
